### DISCUSSION TITLE
/Friends

### DISCUSSION THREAD

**Discussion by giscus** [bosichong/suiyanblog#1] [Announcements]
# Friends 友情链接 | SuiYan 碎言

碎言博客的友情链接

https://www.suiyan.cc/Friends

**Comment by bosichong** [comment]
欢迎互换链接，申请友情链接请先添加本站的链接 ：

名称：碎言
地址：https://www.suiyan.cc/
介绍：坚持深度阅读，持续写作输出，复盘技术实践。以终生学习对抗不确定性，把固执与坚持，All in 在值得的事情上
头像：https://www.suiyan.cc/assets/images/avatar.png
订阅：https://www.suiyan.cc/feed.xml

**Comment by ToLinkNet** [comment]
网站名称: @友友
网址: https://iteuu.com
头像: https://img.erq.cc/biubiu/logo.png
描述: 藏不住了！@友友带你探索有趣的世界。
网站封面: https://img.erq.cc/biubiu/logo-banner.svg

**Reply by bosichong** [reply]
晚一会给你添加，感谢链接。你的博客主题挺漂亮的！

**Reply by ToLinkNet** [reply]
麻烦大哥更新下，哈哈哈感谢

网站名称: 堆叶博客
网址: https://duiye.cn
头像: https://img.erq.cc/biubiu/logo.webp
描述: 「堆叶博客」把知识堆起来
网站封面: https://img.erq.cc/biubiu/banner.webp

**Reply by bosichong** [reply]
已更新

**Reply by ToLinkNet** [reply]
时隔多天，稳定下来了，麻烦大佬更新一下
博客名称：堆新
博客地址：https://duinew.com/
博客描述：堆行成路•新途无界（DUINEW）
博客头像：https://d.duinew.com/logo.webp
横版头像：https://d.duinew.com/logo-w.webp
博客订阅：https://duinew.com/rss.xml

**Reply by bosichong** [reply]
已更新。

**Reply by ToLinkNet** [reply]
哈哈哈 还得麻烦一下大佬 哈哈 这是最后一次了
站名：八对星星
描述：极目星视穹苍无界•足履行者大地有疆
链接：https://8dui.com
图标：https://cf.8dui.com/logo.webp
横标：https://cf.8dui.com/logo-w.webp
订阅：https://8dui.com/rss.xml

**Reply by bosichong** [reply]
已更新

**Comment by zqlit** [comment]
博客很简洁 内容也不错

**Reply by bosichong** [reply]
谬赞了😄

**Comment by YMC0182** [comment]
喜欢这种简洁风格以内容为主，内容也不错。

有的页面内容溢出了，似乎没有强制换行。

**Reply by bosichong** [reply]
嗯，可能是早期发布的，欢迎来访。

**Reply by bosichong** [reply]
有的页面内容溢出了，似乎没有强制换行。  那个页面？可否提供一下？

**Reply by YMC0182** [reply]
> 有的页面内容溢出了，似乎没有强制换行。  那个页面？可否提供一下？

我用的手机，所有文章页x轴滚动条都出来了可以双指缩放，返回顶部按钮应该是和内容边距一样的吧？我看着内容超出按钮了，关于页没有图片也溢出了，应该是内容布局的问题。

**Reply by bosichong** [reply]
确实，我还真没注意，一会有调试一下改改，应该问题不大。感谢🙏
